doc/classes/Transform3D.xml

```diff
@@ -5,7 +5,9 @@
 	</brief_description>
 	<description>
 		A 3×4 matrix (3 rows, 4 columns) used for 3D linear transformations. It can represent transformations such as translation, rotation, and scaling. It consists of a [member basis] (first 3 columns) and a [Vector3] for the [member origin] (last column).
+		?? The [Transform3D] built-in [Variant] type is a 3×4 matrix representing a transformation in 3D space. It contains a [Basis], which on its own can represent rotation, scale, and shearing. Combined with the [member origin], it can also represent a translation.
 		For a general introduction, see the [url=$DOCS_URL/tutorials/math/matrices_and_transforms.html]Matrices and transforms[/url] tutorial.
+		[b]Note:[/b] Godot uses the [b]right-hand coordinate system[/b] (+X is right, +Y is up, and +Z is back). This may be different from the coordinate system you are used to. For example, by convention, 3D assets are rotated towards the camera (+X is [b]left[/b], +Y is up, and +Z is [b]front[/b]). For more information, see the [url=$DOCS_URL/tutorials/assets_pipeline/importing_scenes.html#d-asset-direction-conventions]Importing 3D Scenes[/url] tutorial.
```

Comment:
```suggestion
		[b]Note:[/b] Godot uses the [b]right-hand coordinate system[/b] (+X is right, +Y is up, and +Z is back). This may be different from the coordinate system you are used to. For example, by convention, 3D assets are rotated toward the camera (+X is [b]left[/b], +Y is up, and +Z is [b]front[/b]). For more information, see the [url=$DOCS_URL/tutorials/assets_pipeline/importing_scenes.html#d-asset-direction-conventions]Importing 3D Scenes[/url] tutorial.
```
Minor language quirk, not mandatory, but the added s is generally an [UK English feature](https://en.wiktionary.org/wiki/towards#Usage_notes)

Reply:
A quick search had me notice that there's a whole lot more "towards" than "toward" across the entire class reference (68 vs 18). Maybe this inconsistency needs to be addressed another time, but I can remove the `s` if desired.

Reply:
It's up to you 🙂 (or anyone else more opinionated than me) but just a thought